packages/amplify_core/lib/src/category/amplify_auth_category.dart

```diff
@@ -46,6 +48,68 @@ class AuthCategory extends AmplifyCategory<AuthPluginInterface> {
 
   /// {@template amplify_core.amplify_auth_category.sign_up}
   /// Create a new user with the given [username] and [password].
+  ///
```

Comment:
Might be worth adding a header or something to separate out the examples?

```suggestion
  ///
  /// ## Example
  ///
```